docs/source/getting-started/Setting-up-hypervisor-driver.adoc

```diff
@@ -152,5 +152,17 @@ $ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
 $ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
 ----
 
-For more information, refer to the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
+For more information, see the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
 ====
+
+== Hyper-V
+To use Minishift with Hyper-V:
+
+. Install https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
+. Add a https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch[Virtual Switch].
++
+NOTE: Verify that you pair the virtual switch
+with a __network card (wired or wireless) that is connected to the network__.
```

Comment:
This has been part of our Installation instructions for already 2 months; https://github.com/minishift/minishift/blob/e1b5c98327f42d75b7feee3d4b9ad1078f492ea7/docs/source/getting-started/installing.adoc This originates from the Microsoft documentation: https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network   "Connect to network" which details that to "Connect virtual machines to the internet" you need to do ... such and such, and statng in step 9: "Select the physical network card to be paired with the new virtual switch. This is the network card that is physically connected to the network." We choose not to differ from this